### PR TITLE
add media.emby.EmbyServer

### PR DESCRIPTION
**Pull request by alucryd**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [ ] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

Hopefully that is enough to get accepted. We produce tarballs on github with everything that is needed to build the flatpak. Up until now we provided pre-built flatpaks.

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build media.emby.EmbyServer

File: media.emby.EmbyServer.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "app-id": "media.emby.EmbyServer",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
```
Comment (hfiguiere) [inline media.emby.EmbyServer.json:4]:
```suggestion
    "runtime-version": "20.08",
```

File: media.emby.EmbyServer.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+                    "url": "https://github.com/MediaBrowser/Emby.Releases/releases/download/4.6.2.0/emby-server-flatpak_4.6.2.0_aarch64.tar.xz",
+                    "sha256": "a11b91167f59f00dcbd29367f56570f92136de64343de8c130ee5189250c24b0",
+                    "only-arches": ["aarch64"]
+                },
+                {
+                    "type": "file",
+                    "url": "https://github.com/MediaBrowser/Emby.Releases/releases/download/4.6.2.0/emby-server-flatpak_4.6.2.0_arm.tar.xz",
+                    "sha256": "8ffe780e7fb88e9ee44a2bc337dbc424a4f12b8cca711c160c8e30d8894930dd",
+                    "only-arches": ["arm"]
+                },
```
Comment (hfiguiere) [inline media.emby.EmbyServer.json:25]:
```suggestion
```

Runtime 20.08 shall be used and that's where `arm` has been dropped from supported platforms.

**Review by hfiguiere — CHANGES_REQUESTED** [review]
build fail because of app data validation.

**Review by LeandroStanger — APPROVED** [review]

**Comment by alucryd** [comment]
Bumped the runtime and removed arm, thanks!

**Comment by hfiguiere** [comment]
You didn't address the appdata validation failure.

**Comment by alucryd** [comment]
Apologies, I thought the above issues were it. I'll take care of the rest.

File: media.emby.EmbyServer.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+    "app-id": "media.emby.EmbyServer",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "media.emby.EmbyServer",
+    "finish-args": ["--share=network", "--filesystem=host", "--device=dri"],
+    "modules": [
+        {
+            "name": "media.emby.EmbyServer",
```
Comment (barthalion) [inline media.emby.EmbyServer.json:10]: Nothing wrong with the manifest per se, but: 1) is it actually redistributable? 2) is the conversation about upstream involvement public?

Comment (alucryd) [reply media.emby.EmbyServer.json:10]: It is, there's no public conversation about it, but @LukePulverenti (upstream) can attest to it.

**Comment by LukePulverenti** [comment]
Yes he's working for us and handling this submission.

**Comment by alucryd** [comment]
Sorry about the (very) long delay, I just added a screenshots tag to our appstream file, and bumped the server version. I no longer have access to the full validation log, but I believe that was the last item needed to pass.

**Comment by hfiguiere** [comment]
bot, build media.emby.EmbyServer

**Comment by hfiguiere** [comment]
```
• tag-missing           : <content_rating> required [use https://odrs.gnome.org/oars]
• attribute-missing     : <release> has no timestamp
```

**Comment by alucryd** [comment]
Ah, thanks, will add these then.

**Comment by alucryd** [comment]
Both have been added!

**Comment by hfiguiere** [comment]
bot, build media.emby.EmbyServer

**Comment by alucryd** [comment]
Looks like we're good, tested the build and it works as expected.

**Comment by barthalion** [comment]
/merge @LukePulverenti 

Let us know if we should invite someone more to the Flathub repo.

**Pull request closed without merging**